### DISCUSSION TITLE
babel.config.js not loaded, even though I removed the "babel" config entry in my package.json file

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [shakacode/shakapacker#420] [Q&A]
Any idea why? As I wrote above, i have in my root folder the babel.config.js and removed the "babel" key in the package.json file... also chat gpt cant help me somehow :(

**Comment by justin808** [comment]
Try to create a simple open-source example and reproduce the issue. Sounds like a plain bug in your system.
